src/lib/insights/client.js

```diff
@@ -0,0 +1,54 @@
+export const _inferPayload = (method, results, objectIDs) => {
```

Comment:
should have a public path

Reply:
@Haroenv You mentionned in the review we should expose this function. I would rather not expose it at this point until we get a clearer view of what we should pass to it?
I don't think there is any emergency to expose it, right?